src/platform/amd/common/include/platform/acp_dmic_dma.h

```diff
@@ -14,6 +14,21 @@
 
 /* 109c7aba-a7ba-43c3-b9-42-59-e2-0a-66-11-be */
 
+#define DMIC_SETTLING_TIME_MS 360
```

Comment:
how to justify 360ms for DMIC? based on observation?

Reply:
Yes, based on observation, which is being used for windows environment.